spec/patch/patch.spec.js

```diff
@@ -9,49 +9,49 @@ describe('patch', () => {
-        require('./data/complex-insert-to-middle-without-key'),
-        require('./data/complex-insert-to-ending-with-key'),
-        require('./data/complex-insert-to-ending-without-key'),
-        require('./data/complex-remove-from-beginning-with-key'),
-        require('./data/complex-remove-from-beginning-without-key'),
-        require('./data/complex-remove-from-ending-with-key'),
-        require('./data/complex-remove-from-ending-without-key'),
-        require('./data/complex-reverse'),
-        require('./data/complex-shuffle-with-inserts-removes')
+        require('./data/updateText1').default,
+        require('./data/updateText2').default,
+        require('./data/updateText3').default,
+        require('./data/removeText1').default,
+        require('./data/removeText2').default,
+        require('./data/updateAttr1').default,
+        require('./data/updateAttr2').default,
+        require('./data/updateAttr3').default,
+        require('./data/updateAttr4').default,
+        require('./data/updateAttr5').default,
+        require('./data/updateAttr6').default,
+        require('./data/removeAttr1').default,
+        require('./data/removeAttr2').default,
+        require('./data/removeAttr3').default,
+        require('./data/removeAttr4').default,
+        require('./data/removeAttr5').default,
+        require('./data/replace1').default,
+        require('./data/replace2').default,
+        require('./data/replace3').default,
+        require('./data/replace4').default,
+        require('./data/replace5').default,
+        require('./data/appendChild1').default,
+        require('./data/appendChild2').default,
+        require('./data/removeChild1').default,
+        require('./data/insertChild1').default,
+        require('./data/insertChild2').default,
+        require('./data/moveChild1').default,
+        require('./data/removeChildren1').default,
+        require('./data/complex1').default,
+        require('./data/complex2').default,
+        require('./data/complex3').default,
+        require('./data/complex-insert-to-beginning-with-key').default,
+        require('./data/complex-insert-to-beginning-without-key').default,
+        require('./data/complex-insert-to-middle-with-key').default,
+        require('./data/complex-insert-to-middle-without-key').default,
+        require('./data/complex-insert-to-ending-with-key').default,
+        require('./data/complex-insert-to-ending-without-key').default,
+        require('./data/complex-remove-from-beginning-with-key').default,
+        require('./data/complex-remove-from-beginning-without-key').default,
+        require('./data/complex-remove-from-ending-with-key').default,
+        require('./data/complex-remove-from-ending-without-key').default,
+        require('./data/complex-reverse').default,
+        require('./data/complex-shuffle-with-inserts-removes').default
     ];
 
     data.forEach(specData => {
```

Comment:
My bad, it should be: `{default: specData}`.

[example on babel/repl](http://babeljs.io/repl/#?experimental=false&evaluate=true&loose=false&spec=false&code=var%20data%20%3D%20%5B%20require('qwe')%20%5D%3B%0A%0Adata.forEach((%7Bdefault%3A%20specData%7D)%20%3D%3E%20%7B%0A%20%20%2F%2F%20yo%20%0A%7D)%3B%0A)